src/Elders.Cronus/Discoveries/EventStoreDiscoveryBase.cs

```diff
@@ -14,7 +17,26 @@ protected override DiscoveryResult<IEventStore> DiscoverFromAssemblies(Discovery
         IEnumerable<DiscoveredModel> GetModels(DiscoveryContext context)
         {
             var loadedTypes = context.Assemblies.Find<IEventStoreIndex>();
+            yield return new DiscoveredModel(typeof(EventToAggregateRootId), typeof(EventToAggregateRootId), ServiceLifetime.Transient);
```

Comment:
I would suggest to register all indices here. You could use something like:
```
foreach (var indexType in loadedTypes)
            {
                yield return new DiscoveredModel(indexType, indexType, ServiceLifetime.Singleton);
            }
```

I am 50/50 for the lifestyle singleton/transient. It is safer to go with Transient but leave a comment that we need to revisit it when we add more indices.